build.yaml

```diff
@@ -2258,8 +2280,15 @@ steps:
       export HAIL_TEST_STORAGE_URI={{ global.test_storage_uri }}/{{ token }}
       export HAIL_TEST_RESOURCES_DIR="{{ global.test_storage_uri }}/{{ upload_test_resources_to_blob_storage.token }}/test/resources/"
       export HAIL_DOCTEST_DATA_DIR="{{ global.test_storage_uri }}/{{ upload_test_resources_to_blob_storage.token }}/doctest/data/"
+      export HAIL_GENETICS_QOB_VEP_GRCH37_IMAGE={{ qob_vep_image_grch37.image }}
+      export HAIL_GENETICS_QOB_VEP_GRCH38_IMAGE={{ qob_vep_image_grch38.image }}
       export GOOGLE_APPLICATION_CREDENTIALS=/test-gsa-key/key.json
 
+      if [[ "$HAIL_CLOUD" = "gcp" ]]
+      then
+          export GCS_REQUESTER_PAYS_PROJECT=broad-ctsa
+      fi
```

Comment:
Ya if it's something that the Hail System provides and not the Hail Team provides, I think it should be in our bootstrap process and part of the GCP project that runs the hail system. Worth looking into whether we can get this into terraform but an easier first step might just be adding a script as part of the bootstrap process that uploads this into an appropriate bucket (that is provisioned with terraform). It would be even better if our terraform could use the `http` data provider to do the copy, but I have a feeling that would involve storing the data in the terraform state which sounds not good.

> Regarding (2), I think AUS is using Pulumi. Is it gonna be a pain for them if we start managing infra using terraform but they're using Pulumi?

They run our terraform, but have started using Pulumi for other infrastructural things of theirs. FWIW, I think Pulumi uses terraform, so maybe they use a blend of both.

> In the short term, is our general process this?
Manually make the infra change.
Email AUS & MSFT with a description of the necessary change and the commit that relies on it.

Yes

Reply:
> Worth looking into whether we can get this into terraform but an easier first step might just be adding a script as part of the bootstrap process that uploads this into an appropriate bucket (that is provisioned with terraform).

I agree with Daniel, but I think we should also consider the Terra use case as well. I'll continue to think about this some more.